Example/src/screens/NativeNavigation.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
```

Comment:
I think this screen should not be in the example app since it is a mashup of many different options, and we already have screens for them.